payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncher.kt

```diff
@@ -167,7 +168,11 @@ internal class GooglePayPaymentMethodLauncher internal constructor(
          * has existing payment methods.
          */
         var existingPaymentMethodRequired: Boolean = false
-    ) : Parcelable
+    ) : Parcelable {
+
```

Comment:
Can't this be another parameter of the constructor?

Reply:
I don't want to make this user-customizable.